### PR TITLE
Added IGameServersService Interface with method GetAccountList

### PR DESCRIPTION
**Pull request by mateo9x**

### CODE REVIEW

File: src/main/java/io/github/aquerr/steamwebapiclient/SteamWebApiInterfaceMethod.java [hunk 1]
```diff
@@ -28,7 +28,8 @@
     I_PLAYER_SERVICE_GET_BADGES("IPlayerService", "GetBadges"),
     I_PLAYER_SERVICE_GET_COMMUNITY_BADGE_PROGRESS("IPlayerService", "GetCommunityBadgeProgress"),
 
-    I_STEAM_NEWS_GET_NEWS_FOR_APP("ISteamNews", "GetNewsForApp")
+    I_STEAM_NEWS_GET_NEWS_FOR_APP("ISteamNews", "GetNewsForApp"),
+    I_STEAM_GAME_SERVERS_SERVICE_GET_ACCOUNT_LIST("IGameServersService", "GetAccountList")
```
Comment (Aquerr) [inline src/main/java/io/github/aquerr/steamwebapiclient/SteamWebApiInterfaceMethod.java:32]: I_GAME_SERVERS_SERVICE

Comment (mateo9x) [reply src/main/java/io/github/aquerr/steamwebapiclient/SteamWebApiInterfaceMethod.java]: fixed m8

File: src/main/java/io/github/aquerr/steamwebapiclient/request/AccountListRequest.java [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    @SteamRequestQueryParam("key")
+    @Builder.Default
+    private String key = "";
+
+    @Override
+    public void setApiKey(String apiKey) {}
+
+    @Override
+    public String getApiKey() {
+        return null;
```
Comment (Aquerr) [inline src/main/java/io/github/aquerr/steamwebapiclient/request/AccountListRequest.java:32]: Shouldn't we return the value of `key` field here?

File: src/main/java/io/github/aquerr/steamwebapiclient/request/AccountListRequest.java [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+     * Steamworks Web API user authentication key.
+     * <p>
+     * Added automatically by the client library if not added manually.
+     */
+    @SteamRequestQueryParam("key")
+    @Builder.Default
+    private String key = "";
+
+    @Override
+    public void setApiKey(String apiKey) {}
```
Comment (Aquerr) [inline src/main/java/io/github/aquerr/steamwebapiclient/request/AccountListRequest.java:28]: Shouldn't we set the value of `key` field here?

**Review by Aquerr — APPROVED** [review]

**Pull request merged**